docs/source/advanced_features/use_internal_network.rst

```diff
@@ -0,0 +1,34 @@
+######################
+ Use Internal Network
+######################
+
+This topic we describes how to use the internal network.
+
+Internal Network
+================
+
+What is internal network?
+Take OSS as an example, the uploaded datasets are stored in bucket of different regions,
+the endpoints are the domain names that the other services can use to access OSS.
+see `Endpoints and Regions`_ for more details.
+
+To achieve data acceleration, use internal network if the current region matches the bucket
+region that stores the desired dataset. See *Setting* in `my datasets(or team datasets)`_ panel
+of `TensorBay`_ platform to check the bucket region of dataset.
+
+.. _Endpoints and Regions: https://www.alibabacloud.com/help/doc-detail/31837.htm?spm=a2c63.p38356.879954.10.79f17d9107Wu1o#concept-zt4-cvy-5db
+.. _my datasets(or team datasets): https://gas.graviti.cn/tensorbay/dataset-list
+.. _TensorBay: https://gas.graviti.cn/tensorbay/
+
+Usage
+=====
```

Comment:
Add the following sentences to the top of this part.

For users with unstable network quality, we recommend the following solution:

    # Make sure that a cloud server in a specific region is available.
    # Create and upload datasets to TensorBay storage space which is in the same region as your cloud server.
    # Read the datasets from your cloud server using the internal endpoint.